src/main/java/org/jabref/gui/actions/StandardActions.java

```diff
@@ -32,8 +32,8 @@ public enum StandardActions implements Action {
     REBUILD_FULLTEXT_SEARCH_INDEX(Localization.lang("Rebuild fulltext search index"), IconTheme.JabRefIcons.FILE),
     REDOWNLOAD_MISSING_FILES(Localization.lang("Redownload missing files"), IconTheme.JabRefIcons.DOWNLOAD),
     OPEN_EXTERNAL_FILE(Localization.lang("Open file"), IconTheme.JabRefIcons.FILE, KeyBinding.OPEN_FILE),
-    EXTRACT_FILE_REFERENCES_ONLINE(Localization.lang("Extract references from file (online)"), IconTheme.JabRefIcons.FILE_STAR),
-    EXTRACT_FILE_REFERENCES_OFFLINE(Localization.lang("Extract references from file (offline)"), IconTheme.JabRefIcons.FILE_STAR),
+    EXTRACT_FILE_REFERENCES_ONLINE(Localization.lang("Extract references from file (%0)", Localization.lang("offline")), IconTheme.JabRefIcons.FILE_STAR),
```

Comment:
This is an absolute bad iea because you cannot combine these words in such way in all languages

Reply:
The pattern was to always have `(offline)` as suffix. OK, I can rework this change. 😅